nucleus/__init__.py

```diff
@@ -518,6 +519,73 @@ def create_model(
             client=self,
         )
 
+    def create_launch_model(
+        self,
+        name: str,
+        reference_id: str,
+        bundle_args: Dict[str, any],
```

Comment:
Should we directly expose things required for launch.create_model_bundle{_from_dir}()? (e.g. load_predict_fn/predict_fn/etc.)?
On second thought, that might make maintaining version compatibility really annoying

Reply:
yeah... i need to fix the any for linting but we'd have to make changes twice :/

Reply:
Hm yeah this is interesting... in the process of trying to streamline one particular case, it does bring up the point of possibly writing 2N the number of client functions. Think it's probably fine for now? The nucleus client is just using the launch client's public functions anyway.

Reply:
Maybe another way of thinking about this: the only added value from doing Launch operations inside the Nucleus client is that you don't have to do a separate set of Nucleus operations afterwards. Just a broad thought to keep in mind as we're building this out.